docs/src/pages/components/tabs/demo.tsx

```diff
@@ -1,23 +1,48 @@
 import * as React from 'react';
 import { Tabs, TabItem, Flex, Button, View } from '@aws-amplify/ui-react';
 
-import { TabsPropControls } from '@/components/TabsPropControls';
-import { useTabsProps } from '@/components/useTabsProps';
-import { Example } from '@/components/Example';
+import { Demo } from '@/components/Demo';
+import { TabsPropControls } from './TabsPropControls';
+import { useTabsProps } from './useTabsProps';
+
+const propsToCode = (props) => {
+  return (
+    `<Tabs` +
+    `\n  currentIndex={${props.currentIndex}}` +
+    `${props.spacing ? `\n  spacing={${props.spacing}}` : ``}` +
```

Comment:
Can we demonstrate passing the `spacing`, `justifyContent` and `indicatorPosition` props as strings instead of as variables?

e.g., `spacing="${props.space}"`

Reply:
Oops you're right. Let me fix that.

Reply:
Updated in latest commit.